### PR TITLE
Update README to describe the EP procedure

### PR DESCRIPTION
**Pull request by benjaminrose**
Here is a draft for the README. @kbarbary @djperrefort

Two things:

1. This does not describe what metrics will be used in the decision process. Astropy did that with their first APE. 
2. Do we want DOIs on each EP? I am not sure it is necessary, but I don't think it hurts anything either.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,88 @@
...
+
+8. Get the source for the DOI badge from the newly-created Zenodo record page by
+   clicking on the DOI badge on the right side of the page and copying the
+   markdown source.
+9. On GitHub (or locally) edit ``README.md`` and add an entry for the new EP to the
+   "Accepted EPs" table.  Use the DOI link from the previous step. Add
+   corresponding Markdown link refs for both the DOI link and the new EP. Preview
+   the update and test the links to make sure they are all correct. Then commit
+   directly to master (or PR if you prefer).
+10. Send an email to the [SNcosmo mailing list](mailto:sncosmo@googlegroups.com)
```
Comment (kbarbary) [inline README.md]: The mailing list is not really used and I'm not sure who is even on it. So I'd remove this for the time being.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,88 @@
...
+beyond a single Pull Request. This is intended to mirror the long-standing Python Enhancement 
+Proposal process and Astropy's Proposals for Enhancement, but generally not quite as
+formally as those much larger projects. Normally a proposal goes through various phases of 
+consideration. Discussion is expected to take place using existing mechanisms (github, mailing 
+list, etc), and eventually a decision is made regarding whether the proposal should be accepted, 
+rejected, or tabled for further discussion later.
+
+## Accepted EPs
+
+| # |     Title                          | Date        | DOI   |
```
Comment (kbarbary) [inline README.md]: I think we can leave DOIs off. Like you write in the intro, this is a more informal process than astropy's. Let's keep it as lightweight as possible until we see a need for something more.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,88 @@
...
+that require extensive work that few are willing to perform without some assurance it will be accepted, 
+provisional acceptance is an option. For serious consideration it is usually good to show that detailed 
+technical aspects have been played with in real code rather (even if it isn't a complete implementation).
+ 
+ 
+## Finalizing EPs
+ 
+The final decision on accepting, rejecting, or tabling an EPs lies with [Kyle Barbary](https://github.com/kbarbary). 
+Once the community discussion on the EP has wound down, Kyle makes the
+final decision on acceptance, rejection, or defering action. One of the community members should then:
```
Comment (kbarbary) [inline README.md:38]: Just a note: I hope and expect to find community consensus on any submitted EPs! But this language is good in the case that's not possible.

**Review by kbarbary — APPROVED** [review]
Looks good. I think we can remove the zenodo stuff though. If someone wants to create a DOI they can still do it, but it would be nice if it weren't a requirement for the process.

**Comment by benjaminrose** [comment]
As the commit message says, I removed references to DOIs and the mailing list.

**Pull request merged**